src/actions/PushOperation.ts

```diff
@@ -482,45 +399,49 @@ class PushOperation {
         return increment;
     }
 
-    /**
-     *
-     * @param {Reservation} reservation
-     * @param {AssetVersion[]} assetVersions
-     * @returns {Promise<void>}
-     */
-    async commitReservation(reservation, assetVersions) {
+    async commitReservation(reservation: Reservation, assetVersions: AssetVersion[]): Promise<void> {
         return await this._registryService.commitReservation(reservation.id, assetVersions);
     }
 
-    /**
-     *
-     * @param {Reservation} reservation
-     * @returns {Promise<void>}
-     */
-    async abortReservation(reservation) {
+    async abortReservation(reservation: Reservation): Promise<void> {
         await this._registryService.abortReservation(reservation);
     }
 
-    /**
-     *
-     * @param {AssetDefinition} asset
-     * @return {Promise<AssetReference[]>}
-     */
-    async resolveDependencies(asset) {
+    async resolveDependencies(asset: AssetDefinition): Promise<AssetReference[]> {
         return this._registryService.resolveDependencies(asset);
     }
 
-    /**
-     *
-     * @param {AssetDefinition} asset
-     * @param {ReferenceMap[]} dependencies
-     * @return {Promise<AssetDefinition>}
-     */
-    async updateDependencies(asset, dependencies) {
+    async updateDependencies(asset: AssetDefinition, dependencies: ReferenceMap[]): Promise<AssetDefinition> {
         return this._registryService.updateDependencies(asset, dependencies);
     }
 
-    getReadmeData() {
+    async getAttachments(): Promise<Attachment[]> {
```

Comment:
Looks up 2 specific config files and adds them to attachments array (to be put into the published kapeta.yml